build/config.cmake

```diff
@@ -113,6 +113,7 @@ SET (ESCARGOT_CXXFLAGS_COMMON "${ESCARGOT_CXXFLAGS_COMMON} -Wno-deprecated-decla
 SET (ESCARGOT_CXXFLAGS_COMMON "${ESCARGOT_CXXFLAGS_COMMON} -Wno-implicit-fallthrough")
 SET (ESCARGOT_CXXFLAGS_COMMON "${ESCARGOT_CXXFLAGS_COMMON} -DESCARGOT_ENABLE_TYPEDARRAY")
 SET (ESCARGOT_CXXFLAGS_COMMON "${ESCARGOT_CXXFLAGS_COMMON} -DESCARGOT_ENABLE_PROMISE")
+#SET (ESCARGOT_CXXFLAGS_COMMON "${ESCARGOT_CXXFLAGS_COMMON} -DESCARGOT_ENABLE_PROXY")
```

Comment:
Is it worth adding this line if it's commented out?

Reply:
Proxy is now work-in-progress.
after fully updated, I'll enable the Proxy feature with TCs. Thank you.